tests/net/lib/coap/src/main.c

```diff
@@ -1551,11 +1826,53 @@ static int test_parse_malformed_marker(void)
 	return result;
 }
 
+static int test_token(void)
```

Comment:
Need not to write separate test case for only token. Version, type, code, id, token and etc.. will be there in every coap packet. Verify all those information in one of the test case. That will be enough I guess.

Reply:
Taken care